packages/tofu/src/inspectSesCompat.js

```diff
@@ -54,8 +89,13 @@ function inspectSesCompat(ast) {
   return results
 }
 
+/**
+ *
+ * @param {string[]} memberPath
+ * @returns {boolean}
+ */
 function hasSetterInWhitelist(memberPath) {
-  let allowListTarget = sesAllowlist
+  let allowListTarget = /** @type {Record<string, any>} */ (sesAllowlist)
```

Comment:
there's another way to do this, but don't know if it's worth bothering

Reply:
You know I'm not the best person to ask if typing something properly is worth bothering, but this here is not really propagating anywhere we'd visit often.